ScreenHome.js

```diff
@@ -15,22 +17,46 @@ class ScreenHome extends Component {
     console.log(this.props, "props is here");
     return (
       <View style={styles.container}>
+        <Image
+          source={require('./img/app-nav.png')}
+          style={styles.nav}
+        />
         <TouchableHighlight
           onPress={() => navigate("ScreenSplash", {screen: "Screen Splash"})}
-          style={styles.button}>
-          <Text
-            style={styles.buttonText}>Screen One </Text>
+          style={styles.magic}>
+          <Image
+          source={require('./img/magic-promo.png')}
+          style={styles.magic}
+        />
         </TouchableHighlight>
+        <Image
+          source={require('./img/promo-ad.png')}
+          style={styles.promo}
+        />
+
```

Comment:
Try including the img folder in the repo if you didn't already, or run `git add . ` to make sure it gets included.